### DISCUSSION TITLE
Freeze list for apps.

### DISCUSSION THREAD

**Discussion by sak96** [d4rken-org/sdmaid-se#353] [Ideas]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

> I prefer to keep suspicious app frozen. 
> And only unfreeze when i have to use and freeze it back.

**Describe the solution you'd like**

> I would like it if SD Maid have list of app in freeze list. 
> unfreezing any of the app will put notification to freeze that app

**Describe alternatives you've considered**

> there can be single button which freezes all app in the list.

**Describe why this would be in the interest of all users**
Describe why this is not a niche feature, but would be a benefit for the majority of users.

> The audience of freeze app is usually people 
> who have list of app then want to unfreeze only when required.
> i may be wrong.

**Additional context**
There is a app which already does this i guess.  [FreezeYou](https://github.com/FreezeYou/FreezeYou)
The feature request template is very clean

**Comment by d4rken** [comment]
What's the issue with `FreezeYou`? Does it lack something?

**Comment by sak96** [comment]
No it doesn't. I keep using it.

i had been pro user sdmaid before shifting to fdroid.
wanted to just know about the idea behind freezing apps in sdmaid.
it was always felt like a usecase issue.

Not a priority. but i would wonder if the ui could be in a bit different ?
